### ISSUE TITLE
Multiple Kafka consumers in the same Akka stream graph not working

### ISSUE THREAD

**Issue by robertomoreno** [akka/reactive-kafka#153]
Just trying to create a graph with multiple consumers sources like this:

```scala
val first :: second :: tail = (1 to KAFKA_NUMBER_PARTITIONS).toList.map { i =>

     val consumerSettings = ConsumerSettings(system, new ByteArrayDeserializer, new StringDeserializer, KAFKA_TOPICS)
        .withBootstrapServers(KAFKA_SERVER_URL)
        .withGroupId(KAFKA_GROUP)
        .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
        .withClientId("client" + i)

     Consumer.atMostOnceSource(consumerSettings)
}

Source.combine(first, second, tail: _*)(Merge(_))
      .map(_.value)
      .runForeach(println)
```

But no no messages are consumed and after a seconds this logs are thrown (a number of lines equals to the KAFKA_NUMBER_PARTITIONS - 1):

> [kafka-akka-cassandra-akka.kafka.default-dispatcher-8] INFO org.apache.kafka.clients.consumer.internals.AbstractCoordinator - Marking the coordinator 2147483647 dead.
> [kafka-akka-cassandra-akka.kafka.default-dispatcher-8] INFO org.apache.kafka.clients.consumer.internals.AbstractCoordinator - Marking the coordinator 2147483647 dead.
> [kafka-akka-cassandra-akka.kafka.default-dispatcher-8] INFO org.apache.kafka.clients.consumer.internals.AbstractCoordinator - Marking the coordinator 2147483647 dead.

and logs in kafka broker are:

> 2016-05-26 09:13:44,623] INFO [GroupCoordinator 0]: Stabilized group group1 generation 2 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:13:44,624] INFO [GroupCoordinator 0]: Assignment received from leader for group group1 for generation 2 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:13:44,736] INFO [GroupCoordinator 0]: Preparing to restabilize group group1 with old generation 2 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:14,626] INFO [GroupCoordinator 0]: Stabilized group group1 generation 3 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:14,627] INFO [GroupCoordinator 0]: Assignment received from leader for group group1 for generation 3 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:14,740] INFO [GroupCoordinator 0]: Preparing to restabilize group group1 with old generation 3 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:44,628] INFO [GroupCoordinator 0]: Stabilized group group1 generation 4 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:44,629] INFO [GroupCoordinator 0]: Assignment received from leader for group group1 for generation 4 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:45,172] INFO [GroupCoordinator 0]: Preparing to restabilize group group1 with old generation 4 (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:14:45,172] INFO [GroupCoordinator 0]: Group group1 generation 4 is dead and removed (kafka.coordinator.GroupCoordinator)
> [2016-05-26 09:19:26,249] INFO [Group Metadata Manager on Broker 0]: Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.GroupMetadataManager)

If n streams are runned instance of using a graph all works as expected, like this:

```scala
(1 to KAFKA_NUMBER_PARTITIONS).toList.map { i =>

      val consumerSettings = ConsumerSettings(system, new ByteArrayDeserializer, new StringDeserializer, KAFKA_TOPICS)
        .withBootstrapServers(KAFKA_SERVER_URL)
        .withGroupId(KAFKA_GROUP)
        .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
        .withClientId("client" + i)

     Consumer.atMostOnceSource(consumerSettings)
      .map(_.value)
      .runForeach(println)
    }
```

**Comment by jpeel** [comment]
I feel confident that this is fixed with the recent merging of PR #139, but it needs to be tested.

**Comment by 13h3r** [comment]
Please test against master and reopen if needed
